sktime/datasets/tests/test_uschange.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3 -u
+# coding: utf-8
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["Sebastiaan Koel"]
+__all__ = []
+
+import pandas as pd
+from sktime.datasets import load_uschange
+
+
+def check_dataset(checks, y=None, X=None):
+    """
+    asserts if datasets are loaded correctly
+    ----------
+    checks: dictionary with values to assert against should contain:
+        'columns' : list with column names in correct order,
+        'len_y'   : lenght of the y series (int),
+        'len_X'   : lenght of the X series/dataframe (int),
+        'data_types_X' : dictionary with column name keys and dtype as value,
+        'data_type_y'  : dtype if y column (string)
+    y:  y series of the dataset
+    X:  X dataframe or series from dataset
+    """
+    if y is not None:
+        assert(isinstance(y, pd.Series))
+        assert(len(y) == checks['len_y'])
+        assert(y.dtype == checks['data_type_y'])
+
+    if X is not None:
+        if len(checks['data_types_X']) > 1:
+            assert(isinstance(X, pd.DataFrame))
+        else:
+            assert(isinstance(X, pd.Series))
+
+        assert(X.columns.values.tolist() == checks['columns'])
+
+        for col, dt in checks['data_types_X'].items():
+            assert(X[col].dtype == dt)
+
+        assert(len(X) == checks['len_X'])
+
+
+def test_uschange():
```

Comment:
Could you run this on `load_airline` too? You could use 
```python
@pytest.mark.parametrize("data_loader", [load_airline, load_uschange, ...])
def test_data_loaders(data_loader):
    ...
```
This will iterate over `data_loader` and run a separate test for each one.

Reply:
Of course we would have to iterate over the checks too ... perhaps we could add the `checks` dictionary to the docstring and retrieve it via `data_loader.__doc__`, just a thought

Reply:
Maybe it would be cleaner to just declare the checks dictionary at the top of the module and then you would just be parametrising over the dictionary keys? e.g.:

```python
_CHECKS = {
    "uschange": {
        'loader': load_uschange,
        'len_y': 187,
        ...
    }
}

@pytest.mark.parametrize("dataset", [load_airline, load_uschange, ...])
def test_data_loaders(dataset):
    data_loader = _CHECKS[dataset]["loader"]
    ...
```